chainguard-dev/edu
# Add tutorial on creating github issues from enforce cloudevents

## Type of change
documentation

### What should this PR do?
This PR adds a tutorial showing how to use the [enforce-events]([https://github.com/chainguard-dev/enforce-events](https://github.com/jamonation/enforce-events/tree/update-github-issue-opener)) repository to create Github issues out of Enforce cloudevents.

### Why are we making this change?
refs #229 - part one of that issue (github cf. Jira) is complete here.

### What are the acceptance criteria? 
The tutorial should work in a clean Google Cloud project and Kind cluster.

### How should this PR be tested?
It is fairly involved to set everything up - I tested multiple times with terraform destroy/apply and added screenshots, so it should work technically. Happy to help with any questions/guidance on setting up a test environment.
